File: 709-to-lower-case/709-to-lower-case.cpp
class Solution {
public:
    string toLowerCase(string s) {
        string ans = s;
        int offset = 'a' - 'A'; //for python it was ord('a') - ord('A')
        for(int i=0;i < s.size(); i++){
            if(ans[i] >= 'A' && ans[i] <= 'Z'){
                ans[i] = ans[i] + offset;
            }
        }
        return ans;
        
    }
};

File: 709-to-lower-case/709-to-lower-case.py
class Solution:
    def toLowerCase(self, s: str) -> str:
        ans  ='';
        offset = ord('a') - ord('A')
        for i in s:
            if ord(i) >= ord('A') and ord(i) <= ord('Z'):
                ans += chr(ord(i) + offset)
            else:
                ans += i
        
        return ans
 
    
    
    
    
    
    
    
    
    
    
    
    
    
    

        